src/Elements/Radio.react.js

```diff
@@ -65,7 +74,7 @@ class Radio extends Component {
     }
 
     render() {
-        const { align, children, className, disabled, fluid, id, label, labelClick, name, pill, style, value } = this.props;
+        const { align, children, className, disabled, fluid, id, label, labelClick, name, pill, style, value, multi } = this.props;
```

Comment:
This line is a getting a little long, let's start breaking the props onto their own lines.